Jenkinsfile

```diff
@@ -15,34 +15,36 @@ pipeline {
         }
         stage('bootstrap') {
             steps {
-                sh './bootstrap.sh'
+                sh '''
+                 carthage bootstrap --platform ios
+                '''
             }
         }
         stage('test') {
             steps {
                 dir('SyncIntegrationTests') {
                     sh 'pipenv install'
-                    sh 'pipenv check'
-                    // sh 'pipenv run pytest ' +
-                        //'--color=yes ' +
-                        //'--junit-xml=results/junit.xml ' +
-                        //'--html=results/index.html'
+                    sh 'pipenv check -i 36546' // Ignoring vulnerability
```

Comment:
@davehunt, there is this issue when running `pipenv check`: 

```
36546: requests <=2.19.1 resolved (2.19.1 installed)!
12:32:17 The Requests package before 2.19.1 sends an HTTP Authorization header to an http URI upon receiving a same-hostname https-to-http redirect, which makes it easier for remote attackers to discover credentials by sniffing the network.
```

I had to change the Jenkins file as we did in the past but with this vulnerability	`sh 'pipenv check -i 36546’` to have the tests working.

Would that be fine? Any other way to avoid this issue? Thanks!

Reply:
We should update the requests package. It's failing because the version we're using has a security issue. You can do this by running `pipenv update` on the command line. All packages will be updated and a new `Pipfile.lock` will be generated. It's important to test it locally though. Alternatively, I believe you can update just requests by doing `pipenv update requests`. I usually let dependency management tools take care of this, such as Dependabot. If you'd like to set this up for the repository I'm happy to help.

Reply:
You can leave this in for now, but remember to remove it once you've updated requests.

Reply:
> You can do this by running pipenv update on the command line

Oh sorry, I forgot that. I can try to fix this in this PR doing what you suggest. Thanks!